config/config.go

```diff
@@ -156,6 +156,11 @@ var (
 		RefreshInterval: model.Duration(5 * time.Minute),
 	}
 
+	// DefaultECSSDConfig is the default ECS SD configuration.
+	DefaultECSSDConfig = ECSSDConfig{
+		RefreshInterval: model.Duration(60 * time.Second),
```

Comment:
Hi!

In my opinion, I understand what you say, and I agree that could be better to separate service discovery in its own projects, this has more flexibility, different release cycle, less dependencies in the core... But for example the idea of writing this in a file so prometheus reads it I don't think that is a good approach, having a shared file system dependency is not the best way (In my opinion).

Anyway, I would like to help in any form, for example doing a separate project, experimenting this SD in new ways of interacting with prometheus as a plugin/service or whatever idea or architecture that is in discussion or planned.